doc/manual/src/glossary.md

```diff
@@ -94,6 +186,9 @@
     derivations that are stored in the Nix store. These derivations can
     then be built.
 
+  - [IFD]{#gloss-ifd}\
+    [Import From Derivation](./language/import-from-derivation.md)
+
```

Comment:
Import From Derivation (IFD) can be a "historical term" that redirects to building-dependent evaluation.

Reply:
Let us do the renaming in another pull request to focus on the essence of this one.